namespace cpp Ruyi.SDK.StorageLayer
namespace csharp Ruyi.SDK.StorageLayer
namespace java Ruyi.SDK.StorageLayer
namespace netcore Ruyi.SDK.StorageLayer
namespace rs Ruyi.SDK.StorageLayer

struct GetLocalPathResult {
    1: bool result,
    2: string path,
}


